rotkehlchen/tests/api/test_misc.py

```diff
@@ -518,3 +518,26 @@ def test_cache_deletion(rotkehlchen_api_server):
     )
     assert response.status_code == 200
     assert response.headers['Content-Type'] == 'image/png'
+
+
+@pytest.mark.parametrize('have_decoders', [True])
+def test_query_frontend_mappings(rotkehlchen_api_server):
+    """
+    Test that the information for event types mappings and counterparties is correctly
+    generated for the frontend.
+    """
+    response = requests.get(
+        api_url_for(
+            rotkehlchen_api_server,
+            'typesmappingsresource',
+        ),
+    )
+    assert_proper_response(response)
```

Comment:
you are testing nothing here. Please test at least something or delete the test. This is only testing both endpoints exist.